lib/pymedphys/_version.py

```diff
@@ -1,5 +1,5 @@
 # DO NOT EDIT THIS FILE!
 # This file has been autogenerated by `poetry run pymedphys dev propagate`
 
-version_info = [0, 39, 3]
-__version__ = "0.39.3"
+version_info = [0, 40, 0, "dev0"]
```

Comment:
I think you need to bump to dev2 to be safe...

It appears that there was a 0.40.0-dev1, although I'm not sure if that got packaged and put in to PyPI:
```
commit f8247d17c456cfdd09e1f6bb5b2521b7602e32b5
Author: Simon Biggs <simon.biggs@radiotherapy.ai>
Date:   Thu Oct 6 22:35:33 2022 +1100

    post-release

diff --git a/lib/pymedphys/_version.py b/lib/pymedphys/_version.py
index c9179521..7e1b7833 100644
--- a/lib/pymedphys/_version.py
+++ b/lib/pymedphys/_version.py
@@ -1,5 +1,5 @@
 # DO NOT EDIT THIS FILE!
 # This file has been autogenerated by `poetry run pymedphys dev propagate`
 
-version_info = [0, 39, 2]
-__version__ = "0.39.2"
+version_info = [0, 40, 0, "dev1"]
+__version__ = "0.40.0-dev1"
```

and PyPI definitely already has a 0.40.0-dev0:
<img width="787" alt="Screen Shot 2022-10-30 at 1 58 16 PM" src="https://user-images.githubusercontent.com/16871837/198857994-cb2947d1-c590-4fae-b219-c1f91da65dba.png">

Reply:
@Matthew-Jennings , given the minimal change, I took the liberty of doing it myself, I hope you don't mind.

Reply:
Not at all! Thank you.